lib/node_modules/@stdlib/regexp/to-json/docs/repl.txt

```diff
@@ -0,0 +1,33 @@
+
+{{alias}}( error )
+    Returns a JSON representation of a regexp object.
+
+    The JSON object is guaranteed to have the following properties:
+
+    - type: regexp type.
+    - pattern: regular expression pattern string.
+
+   Depending on the object, the following properties *may* be present:
+
+    - flags: aditional options for matching.
+
+    The function also serializes all enumerable properties.
+
+    Parameters
+    ----------
+    regexp: RegExp
+        Regular expression to serialize.
+
+    Returns
+    -------
+    out: Object
+        JSON representation.
+
+    Examples
+    --------
+    > var regex = new RegExp('ab+c');
```

Comment:
In general, here and elsewhere, we include spaces around function arguments. I.e., `new RegExp( 'ab+c' );`.